libnetwork/osl/interface_linux.go

```diff
@@ -12,6 +13,7 @@ import (
 
 	"github.com/containerd/log"
 	"github.com/docker/docker/internal/nlwrap"
+	"github.com/docker/docker/libnetwork/internal/l2disco"
 	"github.com/docker/docker/libnetwork/ns"
 	"github.com/docker/docker/libnetwork/types"
 	"github.com/pkg/errors"
```

Comment:
We should remove this import in a follow-up, such that we don't need to alias the stdlib `errors` pkg.